indico/modules/rb_new/schemas.py

```diff
@@ -81,6 +84,30 @@ class Meta:
         fields = ('start_dt', 'end_dt', 'is_valid', 'reservation')
 
 
+class ReservationDetailsOccurrenceSchema(mm.ModelSchema):
+    reservation = Nested(ReservationSchema)
+
+    class Meta:
+        model = ReservationOccurrence
+        fields = ('start_dt', 'end_dt', 'is_valid')
+
+
+class ReservationDetailsSchema(mm.ModelSchema):
+    booked_for_user = Nested(UserSchema, only=('full_name', 'phone', 'email'))
+    created_by_user = Nested(UserSchema, only=('full_name'))
```

Comment:
Looks like the string is correct, but in that case remove the `()` around it.

Reply:
I have already changed it to a tuple, so I'll keep it that way.

Reply:
ok, either way is fine -  just wanted to update the comment since i noticed that it was working fine because your code was actually expecting a flat string instead of an object (which is what nested apparently gives you when specifying a string)